ValheimPlus/Utility/ZNetExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿public static class ZNetExtensions
+{
+    public static bool IsLocalInstance(this ZNet znet)
+    {
+        return !znet.IsDedicated() && znet.IsServer() && (znet.GetNrOfPlayers() > znet.GetPeers().Count);
+    }
+
+    public static bool IsClientInstance(this ZNet znet)
+    {
+        return !znet.IsDedicated() && !znet.IsServer();
+    }
+
+    public static bool IsServerInstance(this ZNet znet)
+    {
+        return !znet.IsLocalInstance() && !znet.IsClientInstance();
+    }
+
+    public static string GetInstanceType(this ZNet znet)
+    {
+        if (znet.IsLocalInstance())
+        {
+            return "Local";
```

Comment:
This seems like a perfect spot for an Enum - just define it in the same file under ZNetExtensions